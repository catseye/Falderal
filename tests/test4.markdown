Falderal Test 4
---------------

Another Falderal document which is ill-formed.

    -> Functionality "Cat" is implemented by
    -> shell command "python cat.py"

    -> Tests for functionality "Cat"

Cat cats.

    = meow

Oops, I didn't include a test body.
