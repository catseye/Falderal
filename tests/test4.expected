FalderalSyntaxError: line 13: expectation must be preceded by test body or test input
